formats/service_manual_guide/frontend/examples/service_manual_guide.json

```diff
@@ -11,6 +11,7 @@
         "href": "#what-is-it-why-it-works-and-how-to-do-it"
       }
     ],
+    "change_note": "This is our latest change.\n\nThis is the reason for our latest change.",
```

Comment:
I wonder how much we need the reason for change in here? It's minorly upsetting that the `change_note` field contains the change_note and reason_for_change, but doesn't in the `change_history`.

Considering the actual email isn't the most readable at the moment it might actually improve it and the user can always click through to the frontend to see the reason_for_change.

Already been down this path?